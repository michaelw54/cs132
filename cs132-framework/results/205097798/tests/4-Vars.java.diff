1,4c1,3
< ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
< Was expecting:
<     "return" ...
<     
---
> 12
> 15
> 27
