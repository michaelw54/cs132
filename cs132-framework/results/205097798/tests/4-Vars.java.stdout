func Main()
w_2 = alloc(4)
w_3 = alloc(8)
[w_2 + 0] = w_3
[w_3 + 0] = @A_helper
[w_3 + 4] = @A_run
w_1 = @A_run
w_0 = call w_1(w_2)
print(w_0)
ret_ = 0
return ret_

func A_run(this)
a = 0
b = 0
java.lang.NullPointerException
